geode-apis-compatible-with-redis/README.md

```diff
@@ -63,14 +58,17 @@ redis-cli -h <compatibleWithRedisBindAddress> -p <compatibleWithRedisPort> -a <c
 
 If the server is functioning properly, you should see a response of `PONG`.
 
-### <a name="adding-a-server"></a>Optional - Adding an additional Geode server with compatible with Redis APIS
+[Return to top](#top)
+
+### <a name="adding-a-server"></a>Optional - Adding an additional Geode server with APIs Enabled
```

Comment:
I know each word in this title wasn't capitalized before, but should it be?

Reply:
This should match whatever's chosen for the revision on lines 7 & 8.

Reply:
honestly I'm not sure. I left it the way it was assuming it was correct, but to me it makes more sense to have it capitalized, so I'll do that unless someone from docs disagrees with that choice